src/components/calculator.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+
+// eslint-disable-next-line react/prefer-stateless-function
+class Calculator extends React.Component {
+  // constructor(props) {
+  //   super(props);
+  // }
```

Comment:
## [OPTIONAL] i mark this as optional because the previous reviewer missed this but i will strongly suggest that you remove the inline `eslint disable`  and fix the code this would help you write clean code and cause fewer bugs 🐛  for you. HINT 💭 to get rid of the prefer stateless function error  you can write your class like this `class Calculator extends React.PureComponent` this lets react know your class component is stateless 👍

Reply:
Thank you very much @Davidosky007. I will implement this. I looked for a solution and only found the linter disable.